### ISSUE TITLE
Send reminder 1 hour before scheduled Ensemble

### ISSUE THREAD

**Issue by tedyoung** [jitterted/mobreg#80]
Depends on #78 - include the link in the reminder email

**Comment by tedyoung** [comment]
Split this into two stories:
- [ ] Add a "remind accepted" button on the Admin UI that triggers the reminder email to be sent
- [ ] Try out the cron-based `@Scheduled` support in Spring to check for any Ensembles scheduled to start in the next 62 minutes (one hour plus a bit of slop). For each one it finds (can I assume there's only one??), trigger the code to send the reminder. (Quartz is overkill for this).
